src/components/DatePicker/DatePicker.scss

```diff
@@ -11,6 +11,23 @@ $font-size: rem(12px);
 // stylelint-disable-next-line color-no-hex
 $in-range-border-color: #9ca6de;
```

Comment:
This ❄️ colour will be a problem when different themes are applied. I wonder if we can apply the selected colour at 10% opacity as it's almost identical?